apps/passport-server/src/services/generic-issuance/pipelines/LemonadePipeline.ts

```diff
@@ -350,7 +399,7 @@ export class LemonadePipeline implements BasePipeline {
     return correspondingEventConfig.externalId;
   }
 
-  private eddsaTicketToLemonadeTierId(ticket: EdDSATicketPCD): string {
+  private eddsaTicketToLemonadeTicketTypeId(ticket: EdDSATicketPCD): string {
```

Comment:
this function (and the one above it - `eddsaTicketToLemonadeEventId`) is unused now - delete?